torchaudio/transforms.py

```diff
@@ -155,6 +155,12 @@ class GriffinLim(torch.nn.Module):
             Values near 1 can lead to faster convergence, but above 1 may not converge. (Default: ``0.99``)
         length (int, optional): Array length of the expected output. (Default: ``None``)
         rand_init (bool, optional): Initializes phase randomly if True and to zero otherwise. (Default: ``True``)
+
+        Example
+            >>> batch, freq, time = 2, 257, 100
+            >>> spectrogram = torch.randn(batch, freq, time)
+            >>> transform = transforms.GriffinLim(n_fft=512)
+            >>> waveform = transform(spectrogram)
```

Comment:
The indentations still needs to be fixed. There should be only 4 spaces before ``Example`` and 8 spaces for the rest.
After fixing the indentation, the PR is good to be merged🙂 @harishsdev.